test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ContractMintHTSSuite.java

```diff
@@ -605,9 +605,9 @@ private HapiApiSpec rollbackOnFailedAssociateAfterNonFungibleMint() {
 						getAccountBalance(TOKEN_TREASURY).hasTokenBalance(nonFungibleToken, 0),
 						childRecordsCheck(nestedMintTxn, CONTRACT_REVERT_EXECUTED,
 								recordWith()
-										.status(SUCCESS)
-										.newTotalSupply(1)
-										.serialNos(List.of(1L)),
+										.status(REVERTED_SUCCESS)
```

Comment:
Would it be better if we leave the values of initially successful transactions intact after we revert them to `REVERTED_SUCCESS`? e.g `new total supply` & `serialNos` would be `1` and `1L` but status will be still `REVERTED_SUCCESS`.

I guess that users can access the information of what the actual mint was (input data) from the synthetic transaction?

Reply:
Hmm interesting question...not sure. I think having no visible side effects in records is preferable here.